lib/matplotlib/backends/backend_wx.py

```diff
@@ -1826,9 +1826,56 @@ def remove_rubberband(self, dc=None):
             self._rect = None
 
 
+class _HelpDialog(wx.Dialog):
+    def __init__(self, parent, help_entries, title="Help"):
+        wx.Dialog.__init__(self, parent, title=title,
+                           style=wx.DEFAULT_DIALOG_STYLE | wx.RESIZE_BORDER)
+
+        sizer = wx.BoxSizer(wx.VERTICAL)
+        grid_sizer = wx.FlexGridSizer(0, 3, 8, 6)
+        # create and add the entries
+        widths = [100, 140, 300]
+        bold = self.GetFont().MakeBold()
+        for r, row in enumerate(help_entries):
+            for (col, width) in zip(row, widths):
+                label = wx.StaticText(self, label=col)
+                if r == 0:
+                    label.SetFont(bold)
+                label.Wrap(width)
+                grid_sizer.Add(label, 0, 0, 0)
+        # finalize layout, create button
+        sizer.Add(grid_sizer, 0, wx.ALL, 6)
+        OK = wx.Button(self, wx.ID_OK)
+        sizer.Add(OK, 0, wx.ALIGN_CENTER_HORIZONTAL | wx.ALL, 8)
+        self.SetSizer(sizer)
+        sizer.Fit(self)
+        self.Layout()
+        self.Bind(wx.EVT_CLOSE, self.OnClose)
+        OK.Bind(wx.EVT_BUTTON, self.OnClose)
+
+    def OnClose(self, evt):
+        HelpWx.dlg = None
```

Comment:
That'll reset the class variable, but in HelpWx you're setting the instance variable.

Reply:
Oh, yes, you're right.
I don't want to pass and store a reference to the HelpWx instance in the dialog.
On the other hand, `self.__class__.dlg = _HelpDialog(...)` is not too nice, so a reference might be better.

Reply:
Ah, `HelpWx.dlg = _HelpDialog(...)` would be an option.
This plus a comment would probably be the the best solution.